tests/acl/custom_acl_table/test_custom_acl_table.py

```diff
@@ -0,0 +1,271 @@
+import logging
+import json
+import pytest
+import time
+
+from ptf.mask import Mask
+import ptf.packet as scapy
+
+
+import ptf.testutils as testutils
+from tests.common.utilities import skip_release
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology("t0"), # Only run on T0 testbed
+    pytest.mark.disable_loganalyzer,  # Disable automatic loganalyzer, since we use it for the test
+]
+
+CUSTOM_ACL_TABLE_TYPE_SRC_FILE = "acl/custom_acl_table/custom_acl_table.json"
+CUSTOM_ACL_TABLE_TYPE_DST_FILE = "/tmp/custom_acl_table.json"
+
+ACL_RULE_SRC_FILE = "acl/custom_acl_table/acl_rules.json"
+ACL_RULE_DST_FILE = "/tmp/acl_rules.json"
+
+LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
+LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
+
+
+@pytest.fixture(scope='module', autouse=True)
+def check_release(rand_selected_dut):
+    skip_release(rand_selected_dut, ["201811", "201911", "202012"])
```

Comment:
@bingwang-ms Is it better move skip to tests_condition_marks.yaml? The benefit is 1) save running time, skip it at verify beginning. 2) Avoid other setup errors when running setup fixtures. Then skipped cases will not become failed or error one.
We have a plan to move all skip check to tests_condition_marks.yaml because of above 2 points I mentioned. If we can do it when adding a new  case, I think it could be great.

Reply:
Great suggestion. Updated. Thanks